### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by chuparCh0pper**
outsourced duplicate code into methods, also added a few TODOs

### CODE REVIEW

File: app/src/main/java/com/kamron/pogoiv/MainActivity.java [hunk 1]
```diff
@@ -152,29 +133,19 @@ public void onClick(View v) {
...
                     }
 
+                    // TODO same calculation as in pokefly @line 201
                     radius = (int) Math.round(displayMetrics.heightPixels / 4.3760683); //dpToPx(157); //(int)Math.round(displayMetrics.heightPixels / 4.37606838); //(int)Math.round(displayMetrics.widthPixels / 2.46153846); //585;
                     if (displayMetrics.heightPixels == 1776 || displayMetrics.heightPixels == 960) {
                         radius++;
                     }
 
-                    trainerLevel = Integer.parseInt(((EditText) findViewById(R.id.trainerLevel)).getText().toString());
+                    trainerLevel = Integer.parseInt(etTrainerLevel.getText().toString());
```
Comment (magiepooh) [inline app/src/main/java/com/kamron/pogoiv/MainActivity.java]: `etTrainerLevel.getText()` maybe return string which is not number.

File: app/src/main/java/com/kamron/pogoiv/MainActivity.java [hunk 2]
```diff
@@ -213,6 +183,26 @@ public void onClick(View v) {
         LocalBroadcastManager.getInstance(this).registerReceiver(takeScreenshot, new IntentFilter("screenshot"));
     }
 
+    private String getVersionName() {
+        try {
+            return getPackageManager().getPackageInfo(getPackageName(), 0).versionName;
+        } catch (PackageManager.NameNotFoundException e) {
+            e.printStackTrace();
```
Comment (magiepooh) [inline app/src/main/java/com/kamron/pogoiv/MainActivity.java]: You should use `Log.e`.

Comment (chuparCh0pper) [inline app/src/main/java/com/kamron/pogoiv/MainActivity.java]: This should be changed for all catch-blocks than...
```java
Log.e(TAG, "Explanation of what was being attempted when the exception was thrown", e);
```
will change it later

**Comment by farkam135** [comment]
Sorry wasn't able to get in KitKat support last night would've prevented a lot of unnecessary work. This pull was awesome! Once conflicts are resolved i'll merge it!

**Pull request merged**